cdxev/__main__.py

```diff
@@ -491,6 +503,81 @@ def create_merge_vex_parser(
     return parser
 
 
+# noinspection PyUnresolvedReferences,PyProtectedMember
+def create_vex_parser(
+    subparser: argparse._SubParsersAction,
+) -> argparse.ArgumentParser:
+    parser = subparser.add_parser(
+        "vex",
+        help="Executes commands [list, search, trim, extract] on VEX/SBOM files.",
+    )
+
+    subparsers = parser.add_subparsers(dest="sub_command", required=True)
+
+    list_parser = subparsers.add_parser(
+        "list", help="Returns a list of all vulnerability IDs."
+    )
+    list_parser.add_argument(
+        "--scheme",
+        help="Set scheme of return list.",
+        choices=["default", "lightweight"],
+        default="default",
+        type=str,
+    )
+
+    list_parser.add_argument(
+        "--format",
+        help="Set format of return file.",
+        choices=["txt", "csv"],
+        default="csv",
+        type=str,
+    )
+
+    search_parser = subparsers.add_parser("search", help="Get vulnerabilities by ID.")
+    search_parser.add_argument(
+        "vul_id",
+        metavar="<vul_id>",
+        help="The ID of the vulnerability to search for.",
+        type=str,
+    )
+
+    trim_parser = subparsers.add_parser(
+        "trim",
+        help="Trims a VEX to show only the vulnerabilities according to the selected state.",
```

Comment:
Why trim is only depending on the `state`? Why not also e.g. on `severity`, maybe I want to only have vulnerabilities with critical severity.